article/app/views/fragments/immersiveGarnettBody.scala.html

```diff
@@ -0,0 +1,83 @@
+@(amp: Boolean = false)(implicit model: ArticlePage, request: RequestHeader, context: _root_.model.ApplicationContext)
+
+@import views.html.fragments.langAttributes
+@import common.LinkTo
+@import views.BodyCleaner
+@import views.support.RenderClasses
+@import views.support.Commercial.{isPaidContent, articleAsideOptionalSizes, shouldShowAds}
+@import views.support.TrailCssClasses.toneClass
+@import experiments._
+@import _root_.model.ContentDesignType.RichContentDesignType
+
+@defining(model.article) { article =>
+  @defining(isPaidContent(model)) { isPaidContent =>
+        <article id="article" data-test-id="article-root"
+
+            class="@RenderClasses(Map(
+                "has-feature-showcase-element" -> (article.tags.isFeature && article.elements.hasShowcaseMainElement),
```

Comment:
Does immersive do something different in a showcase now?

Reply:
🤔

Reply:
@paperboyo No...this was just something I missed and should have removed.